app/Views/welcome_message.php

```diff
@@ -309,7 +331,8 @@
 
 <!-- SCRIPTS -->
 
-<script>
+<script {csp-script-nonce}>
```

Comment:
```suggestion
<script <?= csp_script_nonce() ?>>
```

Reply:
thanks @kenjis , but why dont use  {csp-script-nonce} ? 
like line 12 welcome_message.php,  use {csp-style-nonce}

Reply:
Okay, you are correct. The line 12 uses `<style {csp-style-nonce}>`.
So using it here is okay.

I personally think we don't need to use `{csp-style-nonce}` if we can use the `csp_script_nonce()` function. It is a CI4 global function and we can always use it in PHP scripts.

Reply:
i think so too..
if csp is enable ,it  must following change the placeholder tagNonce , cause If not change and if an attacker injects a string like <script {csp-script-nonce}> it might become the real nonce attribute with functionality.
https://www.codeigniter.com/user_guide/outgoing/csp.html

so the best we can use csp_script_nonce()